Pinta.Core/Classes/Re-editable/Text/TextLayout.cs

```diff
@@ -57,15 +57,14 @@ public TextLayout ()
 	public ImmutableArray<RectangleI> GetSelectionRectangles ()
 	{
 		var regions = engine.SelectionRegions;
-		var rects = new List<RectangleI> ();
-
+		var rects = ImmutableArray.CreateBuilder<RectangleI> (regions.Length);
 		foreach (var region in regions) {
 			PointI p1 = TextPositionToPoint (region.Key);
 			PointI p2 = TextPositionToPoint (region.Value);
-			rects.Add (new RectangleI (p1, new Size (p2.X - p1.X, FontHeight)));
+			RectangleI regionRectangle = new (p1, new Size (p2.X - p1.X, FontHeight));
```

Comment:
Is the extra variable necessary? I think the line looked fine as-is

Reply:
I think it's a matter of preference :) I just pushed a commit to remove the intermediate variable